.circleci/config.yml

```diff
@@ -8,15 +8,18 @@ executors:
   test-executor:
     resource_class: small
     docker:
-      - image: cimg/ruby:3.2.2-browsers
+      - image: checkclientqualifiesdocker/welcome-to-docker:latest
```

Comment:
can we change this `welcome-to-docker` bit and call it something like `circleci_image`?

Reply:
@willc-work now updated 👍🏽